_playwright-tests/UI/GPG_keys.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist, getRowByNameOrUrl } from './helpers/helpers';
+
+export const repoNamePrefix = 'GPG-key';
+export const randomName = () => (Math.random() + 1).toString(36).substring(2, 6);
+export const repoName = `${repoNamePrefix}-${randomName()}`;
+
+export const url = 'https://jlsherrill.fedorapeople.org/fake-repos/signed/';
+export const packages_key =
+  'http://jlsherrill.fedorapeople.org/fake-repos/needed-errata/RPM-GPG-KEY-dummy-packages-generator';
+export const meta_key = 'https://jlsherrill.fedorapeople.org/fake-repos/signed/GPG-KEY.gpg';
```

Comment:
none of these consts should be exported, unless you are using them in other test files and are generic enough 💭 (in that case, they should be moved to helpers)
this seems like test data for only this test, so no need to export/make them public 😅

Reply:
OK, those likes are test specific , will remove the `export`